.vsts-ci/linux.yml

```diff
@@ -80,26 +80,19 @@ jobs:
   - template: ../tools/releaseBuild/azureDevOps/templates/insert-nuget-config-azfeed.yml
 
   - powershell: |
-      tools/travis.ps1 -Stage Bootstrap
+      Import-Module .\tools\ci.psm1
+      Invoke-LinuxTests
     displayName: Bootstrap
     condition: succeeded()
 
   - powershell: |
       $ErrorActionPreference = 'continue'
-      tools/travis.ps1 -NugetKey $(NUGET_KEY)
+      Import-Module tools/ci.psm1
+      $env:NugetKey = $(NUGET_KEY)
+      Invoke-LinuxTests
     displayName: Build and test
     condition: succeeded()
 
-  - powershell: |
-      tools/travis.ps1 -Stage Failure
-    displayName: After Failure
-    condition: failed()
-
-  - powershell: |
-      tools/travis.ps1 -Stage Success
-    displayName: After Success
-    condition: succeeded()
-
   # Uploads any packages as an artifact
   - powershell: |
       Get-ChildItem -Path *.rpm, *.deb, *.tar.gz, TestPackage.zip -Recurse | Select-Object -ExpandProperty FullName | ForEach-Object {
```

Comment:
```suggestion
       Get-ChildItem -Path ${env:BUILD_ARTIFACTSTAGINGDIRECTORY} -Include *.nupkg, *.rpm, *.deb, *.tar.gz, TestPackage.zip -Recurse | Select-Object -ExpandProperty FullName | ForEach-Object {
```

Reply:
It says outdated suggestions can't be applied